src/main/openapi/ds-datahandler-openapi_v1.yaml

```diff
@@ -251,9 +251,13 @@ components:
         password:
           type: string
           description: 'Password if the OAI server require basic authentication'                
-        origin:
+        datasource:
           type: string
-          description: 'Name of the origin when ingesting into DS-storage. The origin must be configured in ds-storage.'
+          description: |
+            Name of the origin when ingesting into DS-storage. The datasource must be configured in ds-storage.
```

Comment:
Does "Name of the origin when ingesting into DS-storage" still hold true?

Reply:
Almost, In line 258-59 I've tried to elaborate. Should I turn the text around?